sympy/combinatorics/subsets.py

```diff
@@ -172,7 +172,7 @@ def next_lexicographic(self):
             else:
                 indices.remove(i)
                 i = i - 1
-                while not i in indices and i >= 0:
+                while i >= 0 and i not in indices:
```

Comment:
Here, I figured it would be faster to do the relational than to check for containment, so put it first.

Reply:
I think `i` is guaranteed to be `int` so this would be faster. It would also be better if `indices` was a `set`.